components/admin/users/user-form.tsx

```
@@ -247,23 +253,25 @@ export default function UserForm({ user, onSuccess, isSubmitting = false, onCanc
             placeholder={t('FULL_NAME_PLACEHOLDER')}
             value={formData.name}
             onChange={(e) => handleInputChange('name', e.target.value)}
             disabled={isSubmittingForm}
             required
-            variant="bordered"
+            variant='bordered'
           />
         </div>
 
         <div>
           <label className="block text-sm font-medium mb-2">{t('TITLE_FIELD')}</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)